src/unittests/f32_interval.spec.ts

```diff
@@ -764,16 +765,22 @@ g.test('asinInterval')
     // prettier-ignore
     [
       // Some of these are hard coded, since the error intervals are difficult to express in a closed human readable
-      // form due to the inherited nature of the errors.
+      // form due to the irrational nature of the values.
```

Comment:
nit:  "irrational" actually means something specific for numbers, and not what you mean here. Do you mean something like "non-sensical"?

Reply:
This is a bit of legacy cruft from the original rewrite, I meant irrational in reference to the values were derived from Pi, since they were no longer inherited, but absolute errors

I am just going to change this to be 'due to the complexity of their derivation', which will cover all it being the span of the intervals from an absolute error and an inherited error, and around Pi.